CliClient/tests/models_Setting.js

```diff
@@ -31,4 +31,19 @@ describe('models_Setting', function() {
 		expect('username' in output).toBe(false);
 	}));
 
+	it('should save values', asyncTest(async () => {
+		await setupDatabase();
+
+		let key = 'sync.5.username';
+		let value = 'testUser';
+		await Setting.setValue(key, value);
```

Comment:
setValue is not async

Reply:
Sorry, that was a mistake. I'll rewrite the tests.